data/human/fleets.txt

```diff
@@ -3357,3 +3357,163 @@ fleet "Human Wormhole Guards"
 		"Frigate (Mark II)"
 	variant 5
 		"Rainmaker (Mark II Old Weapons)"
+
+# The New Tortugans have access to Syndicate hulls, and any Pirate tech,
+# and more varied objectives, so their ships are a quite a mix.
+fleet "Independent (Korath-Allied) Merchants"
+	government "Independent (Korath-Allied)"
+	names "civilian"
+	cargo 3
+	personality
+		timid frugal opportunistic
+		confusion 20
+	variant 10
+		"Bulk Freighter (Blaster)"
+		"Aerie"
+		"Dagger"
+		"Lance"
+	variant 3
+		"Mule (Heavy)"
+		"Manta (Proton)"
+		"Dagger" 2
+	variant 3
+		"Behemoth (Heavy)"
+		"Bastion (Laser)"
+	variant 7
+		"Blackbird"
+		"Quicksilver (Proton)"
+		"Quicksilver"
+		"Raven (Afterburner)"
+		"Blackbird" 2
+	variant 3
+		"Falcon (Heavy)"
+		"Freighter (Proton)"
+		"Freighter"
+	variant 3
+		"Argosy (Blaster)"
+		"Argosy (Laser)"
+		"Argosy (Turret)"
+	variant 7
+		"Behemoth (Speedy)"
+		"Clipper (Speedy)"
+		"Corvette (Speedy)"
+	variant 10
+		"Bulk Freighter (Heavy)"
+		"Firebird (Plasma)"
+		"Fury (Bomber)" 3
+	variant 2
+		"Hauler III"
+		"Leviathan (Particle Chaser)"
+		"Headhunter (Particle)" 2
+	variant 2
+		"Star Queen"
+		"Protector (Proton)"
+		"Splinter (Proton)"
+	variant 7
+		"Aerie"
+		"Dagger" 2
+		"Bounder" 2
+		"Fury (Missile)" 2
+
+# Asteroids beyond Alcyone are often much more durable, so the
+# ordinary mining ships are insufficient.
+fleet "Independent (Korath-Allied) Miners"
```

Comment:
Do you think any of the new ships added in the mining update could be used here?

Reply:
The human mining fleets would be sneezed in half if they were attacked. Normally, putting mining ships in pirate space serves only to show how hopeless it is to be a civilian in pirate space. These ships should be able to hold their own against attacks. Korath and Pirates won't actually attack them, but they'll still have to contend with hostile Syndicate ships.